docs/eso/eso.rst

```diff
@@ -223,7 +223,25 @@ is identical to :meth:`~astroquery.eso.EsoClass.query_instrument`.
 
 ESO instruments without a specific query interface can be queried with
 :meth:`~astroquery.eso.EsoClass.query_main`, specifying the ``instrument`` constraint.
-This is the case of e.g. ``harps`` or ``feros``.
+This is the case of e.g. ``harps``, ``feros`` or the all sky cameras APICAM and MASCOT. Here is an example to query all-sky images from APICAM with ``luminance`` filter. 
+
+.. code-block:: python
+
+    >>> table = eso.query_main(column_filters={'instrument': 'APICAM', 'filter_path': 'LUMINANCE', 'stime':'2019-04-26', 'etime':'2019-04-27'}, cache=False)
+    >>> print(len(table))
+    50
```

Comment:
This is the default limit, it may be beneficial for the example to change it somewhere before the first query example at the top of the page.
To change it to unlimited use `eso.ROW_LIMIT = -1`